cmd/antrea-agent/agent.go

```diff
@@ -320,30 +340,47 @@ func run(o *Options) error {
 		)
 	}
 
-	var mcRouteController *mcroute.MCRouteController
+	var mcDefaultRouteController *mcroute.MCDefaultRouteController
 	var mcStrechedNetworkPolicyController *mcroute.StretchedNetworkPolicyController
+	var mcPodRouteController *mcroute.MCPodRouteController
 	var mcInformerFactory mcinformers.SharedInformerFactory
+	var clusterServiceCIDR *net.IPNet
+	var mcInformerFactoryWithOption mcinformers.SharedInformerFactory
+
+	notEncapMode := networkConfig.TrafficEncapMode != config.TrafficEncapModeEncap
```

Comment:
Do we need the PodRouteController for hybrid mode? If not, we should check `!SupportsEncap()` here.

Reply:
Yes, we need PodRouteController for hybrid mode since the exiting Node route controller will skip installing Pod CIDR route for tunnel interface when the other Node is not in the same subnet of the Node where the agent is running.